packages/veritone-react-common/src/components/Parts/Buttons/PillButton/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
```

Comment:
Kinda surprised to see PillButton in here if we've already done face and object.  Shouldn't it be using the same thing?